.vib/odoo/cypress/cypress/fixtures/companies.json

```diff
@@ -0,0 +1,6 @@
+{
+  "newCompany": {
+    "name": "Thornfield",
+    "phone": "0054326557"
```

Comment:
I think this is not necessary; you are not using `newCompany.phone` in the tests.